web/src/components/Features.js

```diff
@@ -60,87 +65,109 @@ const TextLink = ({ url, label }) => (
   </Link>
 )
 
+const { useGreater } = BreakPointHooks(breakpointsTailwind)
+
 const Features = () => {
-  return (
-    <SectionContainer className='space-y-16 lg:py-18'>
-      <dl className='grid grid-cols-1 lg:grid-cols-4 md:gap-16 lg:gap-x-8 xl:gap-x-16'>
 
+  const isGreaterMd = useGreater('md')
+
+  return (
+    <SectionContainer className="space-y-16 lg:py-18">
+      <dl className="grid grid-cols-1 lg:grid-cols-3 md:gap-8 lg:gap-x-8 xl:gap-x-16">
         <Feature
           Icon={Star}
-          title='Open Source'
-          url='https://github.com/wasp-lang/wasp'
+          title="Open Source"
+          url="https://github.com/wasp-lang/wasp"
           description={`
             This is the way. Wasp is fully open-source and you're welcome to contribute!
           `}
         />
 
         <Feature
           Icon={Unlock}
-          title='Full-stack Auth'
-          url='/blog/2023/04/12/auth-ui'
+          title="Full-stack Auth"
+          url="/blog/2023/04/12/auth-ui"
           description={`
             Add login with social providers or email in a few lines of code with powerful UI helpers. No third party vendor lock-in.
           `}
         />
 
         <Feature
           Icon={Link2}
-          title='RPC (Client <-> Server)'
-          url='/docs/data-model/operations/overview'
+          title="RPC (Client <-> Server)"
+          url="/docs/data-model/operations/overview"
           description={`
             Wasp provides a typesafe RPC layer that instantly brings your data models and server logic to the client.
           `}
         />
 
-        <Feature
-          Icon={Send}
-          title='Simple Deployment'
-          url='/docs/advanced/deployment/overview'
-          description={`
-            Deploy your app to any platform. Wasp offers CLI helpers for the most popular options.
-          `}
-        />
+        <div className="self-center">
+          <Feature
+            Icon={Send}
+            title="Simple Deployment"
+            url="/docs/advanced/deployment/overview"
+            description={`
+              Deploy your app to any platform. Wasp offers CLI helpers for the most popular options.
+            `}
+          />
+        </div>
 
+        <div className="hidden lg:grid place-content-center">
+          <div
+            style={{
+              width: "300px",
+              height: "300px",
+            }}
+          >
+            {isGreaterMd && (
+              <Suspense fallback={null}>
+                <DaBoi height={300} width={300} />
+              </Suspense>
```

Comment:
I don't think boi fits well here, it messes up the page a bit too much in my opinion.

![image](https://github.com/wasp-lang/wasp/assets/1536647/4efb4534-21ac-49cf-83a2-2363327ca585)

What about putting it up in the navbar instead?

![image](https://github.com/wasp-lang/wasp/assets/1536647/68408087-8fb4-4291-9025-2db7ab0572aa)